example/lib/pages/basic_playback.dart

```diff
@@ -25,6 +25,7 @@ class _BasicPlaybackState extends State<BasicPlayback> {
       key: Env.bitmovinPlayerLicenseKey,
       playbackConfig: PlaybackConfig(
         isAutoplayEnabled: true,
+        isBackgroundPlaybackEnabled: true,
```

Comment:
Sidenote: When you add support to configure background playback from dart side, it would be good to have a separate sample for it to not bloat basic playback

Reply:
Agree yes 👍 Will keep that in mind.